src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import 'containers/shared/css/nested-menu.scss'
+import 'containers/Accounts/AccountHeader/styles.scss'
+import 'containers/Accounts/AccountHeader/balance-selector.scss'
+import { AmmDataType } from '../index'
+import { localizeNumber, localizeBalance } from '../../../../shared/utils'
+import Currency from '../../../../shared/components/Currency'
+
+const AMMAccountHeader = (props: AmmDataType) => {
+  const {
+    ammId,
+    balance,
+    balance2,
+    tradingFee,
+    lpBalance,
+    accountId,
+    language,
+    tvl,
+  } = props
+  const { t } = useTranslation()
+  const b1 = localizeBalance(balance, language)
+  const b2 = localizeBalance(balance2, language)
+  const tradingFeeTotal = 1000
```

Comment:
Rename `tradingFeeTotal` to `TRADING_FEE_TOTAL` and move it outside of this component (after imports).

Reply:
done